cardano-config/src/Cardano/TracingOrphanInstances/Consensus.hs

```diff
@@ -5,6 +5,7 @@
 {-# LANGUAGE NamedFieldPuns        #-}
 {-# LANGUAGE OverloadedStrings     #-}
 {-# LANGUAGE ScopedTypeVariables   #-}
+{-# LANGUAGE StandaloneDeriving    #-}
```

Comment:
Was `StandaloneDeriving` needed to remove an import?

Reply:
Thanks for picking up on this. That pragma is not necessary, I probably left it there by accident when I was shuffling around things.